src/main/java/appeng/integration/modules/NEIHelpers/NEIGuiHandler.java

```diff
@@ -0,0 +1,22 @@
+package appeng.integration.modules.NEIHelpers;
+
+import appeng.client.gui.implementations.GuiMEMonitorable;
+import codechicken.nei.api.INEIGuiAdapter;
+import net.minecraft.client.gui.inventory.GuiContainer;
+import net.minecraft.item.ItemStack;
+
+public class NEIGuiHandler extends INEIGuiAdapter {
+    @Override
+    public boolean handleDragNDrop(GuiContainer gui, int mousex, int mousey, ItemStack draggedStack, int button) {
+        if (gui instanceof GuiMEMonitorable && draggedStack != null && draggedStack.getItem() != null)
+        {
+            GuiMEMonitorable gmm = (GuiMEMonitorable)gui;
+            if (gmm.isOverSearchField(mousex, mousey))
+            {
+                gmm.setSearchString(draggedStack.getDisplayName(), true);
```

Comment:
Would use suggest full words for a name, it's much more  readable then acronyms. You don't need to use full name, som thing like `monitorableUI` would be sufficient